l10n_fi_edicode/models/res_partner_operator_einvoice.py

```diff
@@ -0,0 +1,14 @@
+from odoo import fields, models
+
+
+class ResPartnerOperatorEinvoice(models.Model):
+    _name = 'res.partner.operator.einvoice'
+
```

Comment:
`2018-12-14 08:37:59,818 157 WARNING openerp_test odoo.models: The model res.partner.operator.einvoice has no _description` shows up in Runbot. Please add a description for that model.

Reply:
I added description.